code/modules/assembly/flash.dm

```diff
@@ -81,7 +80,6 @@
 
 /obj/item/assembly/flash/examine(mob/user)
 	. = ..()
-	. += "[bulb ? "The bulb looks like it can handle just about [bulb.charges_left] more uses.\nIt looks like you can cut out the flashbulb with a pair of wirecutters." : "The device has no bulb installed."]"
```

Comment:
Just take out the "The bulb looks like it can handle just about [bulb.charges_left] more uses.". Leave the rest as is.